tests/lib/rules/indent.js

```diff
@@ -40,6 +40,25 @@ eslintTester.addRuleTest("lib/rules/indent", {
     valid: [
         {
             code:
+            "if (1 < 2){\n" +
+            "//hi sd \n" +
```

Comment:
I am not sure as this was never part of indent concept from the beginning as far as I understand.